package.json

```diff
@@ -12,7 +12,7 @@
     "lint": "eslint delegated-events.js test",
     "bootstrap": "git submodule update --init && npm install",
     "prebuild": "npm run lint",
-    "build": "rollup -c rollup.config.dist.js",
+    "build": "rollup -c rollup.config.dist.js && cp delegated-events.js.flow dist/",
```

Comment:
This places the type definitions file next to the compiled AMD version of the code. The type definitions are tied to the custom AMD `module.exports` object, so they don't directly apply to the native JavaScript module functions.

Long term, the AMD compilation phase won't be necessary and the native module file can be used directly. Then the type definitions can apply without `module.exports`. So this branch makes a good compromise with the AMD wrapper for now.